File: src/game_of_life/csharp_coderetreat_socramob/gameoflife2_mtest/gameoflife/GameOfLife.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace gameoflife
{
    public class GameOfLife
    {
        private int anzahlNachbarn;

        public GameOfLife(int anzahlNachbarn)
        {
            this.anzahlNachbarn = anzahlNachbarn;
        }

        public bool ZelleUeberlebt()
        {
            bool zelleUeberlebt = (this.anzahlNachbarn >= 2);
            return zelleUeberlebt;
        }

        static void Main(string[] args)
        {
        }
    }
}

File: src/game_of_life/java_coderetreat_socramob/cr_socramob00/GameOfLife.java

public class GameOfLife {

	private boolean istLebendig = false;
	private int anzahlNachbarn = 0;
	
	GameOfLife(int anzahlNachbarn, boolean istLebendig) {
	this.istLebendig = istLebendig;
	this.anzahlNachbarn = anzahlNachbarn;
	}
	
	public boolean istLebendigInNaechsterRunde() {
		return ( this.istLebendig && (this.anzahlNachbarn >=2 && this.anzahlNachbarn<= 3 ));
	}
}
